crates/noirc_evaluator/src/lib.rs

```diff
@@ -114,6 +114,10 @@ impl Evaluator {
         self.current_witness_index
     }
 
+    pub fn push_gate(&mut self, gate: AcirOpcode) {
```

Comment:
```suggestion
    pub fn push_opcode(&mut self, gate: AcirOpcode) {
```

I'd like to ensure that we use "opcode" over "gate" going forwards.

Reply:
Done